src/pt/spectralscan/src/eu/kanade/tachiyomi/extension/pt/spectralscan/NexusScan.kt

```diff
@@ -0,0 +1,311 @@
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    // ==================== Chapter =======================
+
+    private fun getMangaSlug(url: String) = url.substringAfter("/manga/").trimEnd('/')
+
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        return client.newCall(chapterListRequest(manga)).asObservableSuccess()
+            .map { it.parseAs<ChapterListResponse>() }
+            .flatMap { firstPage ->
+                val chapters = parseChaptersFromHtml(firstPage.chapters_html)
+
+                if (!firstPage.has_next) {
+                    Observable.just(chapters)
+                } else {
+                    fetchRemainingChapters(getMangaSlug(manga.url), chapters, 2)
+                }
+            }
+    }
+
+    private fun fetchRemainingChapters(
+        slug: String,
+        accumulatedChapters: List<SChapter>,
+        currentPage: Int,
+    ): Observable<List<SChapter>> {
+        val url = "$baseUrl/ajax/load-chapters/".toHttpUrl().newBuilder()
+            .addQueryParameter("item_slug", slug)
+            .addQueryParameter("page", currentPage.toString())
+            .addQueryParameter("sort", "desc")
+            .addQueryParameter("q", "")
+            .build()
+
+        return client.newCall(GET(url, headers)).asObservableSuccess()
+            .map { it.parseAs<ChapterListResponse>() }
+            .flatMap { page ->
+                val allChapters = accumulatedChapters + parseChaptersFromHtml(page.chapters_html)
+
+                if (page.has_next) {
+                    fetchRemainingChapters(slug, allChapters, currentPage + 1)
+                } else {
+                    Observable.just(allChapters)
+                }
+            }
+    }
+
+    private fun parseChaptersFromHtml(html: String): List<SChapter> {
+        val document = Jsoup.parse(html, baseUrl)
+        return document.select("a.chapter-item[href]").map { element ->
+            SChapter.create().apply {
+                name = element.selectFirst(".chapter-number")?.text()?.trim()
+                    ?: element.attr("data-chapter-number")?.let { "Capítulo $it" }
+                    ?: "Capítulo"
+                setUrlWithoutDomain(element.absUrl("href").ifEmpty { element.attr("href") })
+
+                element.selectFirst(".chapter-date")?.text()?.let { dateStr ->
+                    date_upload = parseChapterDate(dateStr)
+                }
+            }
+        }
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        val slug = getMangaSlug(manga.url)
+        val url = "$baseUrl/ajax/load-chapters/".toHttpUrl().newBuilder()
+            .addQueryParameter("item_slug", slug)
+            .addQueryParameter("page", "1")
+            .addQueryParameter("sort", "desc")
+            .addQueryParameter("q", "")
+            .build()
+        return GET(url, headers)
```

Comment:
Don't you also need `ajaxHeaders()` here? You're using the `/ajax/` endpoint.
```suggestion
        return GET(url, ajaxHeaders())
```